src/lib/utils/hooks/SynapseAPI/useUserBundle.ts

```diff
@@ -3,14 +3,38 @@ import { SynapseClient } from '../..'
 import { getProfilePic, UserProfileAndImg } from '../../functions/getUserData'
 import { SynapseClientError } from '../../SynapseClient'
 import { useSynapseContext } from '../../SynapseContext'
-import { UserProfile } from '../../synapseTypes'
+import { NotificationEmail, UserProfile } from '../../synapseTypes'
+
+export function useGetNotificationEmail(
+  options?: UseQueryOptions<NotificationEmail, SynapseClientError>,
+) {
+  const { accessToken } = useSynapseContext()
+  return useQuery<NotificationEmail, SynapseClientError>(
+    ['notificationEmail'],
+    () => SynapseClient.getNotificationEmail(accessToken),
+    options,
+  )
+}
+
+export function useGetCurrentUserProfile(
+  options?: UseQueryOptions<UserProfile, SynapseClientError>,
+) {
+  const { accessToken } = useSynapseContext()
+  const queryKey = ['user', 'current', 'profile']
+
+  return useQuery<UserProfile, SynapseClientError>(
+    queryKey,
+    () => SynapseClient.getUserProfile(accessToken),
+    options,
+  )
+}
 
 export function useGetUserProfile(
   principalId: string,
   options?: UseQueryOptions<UserProfile, SynapseClientError>,
 ) {
   const { accessToken } = useSynapseContext()
-  const queryKey = [accessToken, 'user', principalId, 'profile']
+  const queryKey = ['user', principalId, 'profile']
```

Comment:
In general, we should drop the `accessToken` from react-query keys. It makes debugging the cache much harder, and the cache should be cleared when the user logs out anyways.